docs/source/contributing/developing-core.md

```diff
@@ -104,15 +104,16 @@ For the fish shell, see [nvm.fish](https://github.com/jorgebucaran/nvm.fish).
 1.  Install or update the supported LTS versions of Node.js, then activate the version supported in Volto.
 
     ```shell
-    nvm install "lts/*"
-    nvm use 20
+    nvm install 21.5.0
+    nvm use 21.5.0
```

Comment:
Revert the above two lines. Volto only supports LTS versions, and 21.x is not LTS.